src/tests/test_scripts/log_meta_test.py

```diff
@@ -81,7 +81,7 @@ def meta_nightly(options, addln_opts):
     subprocess.check_call(options.dirpath + "test_meta_blk_mgr " + cmd_opts + addln_opts, stderr=subprocess.STDOUT,
                           shell=True)
 
-    cmd_opts = "--min_write_size=10485760 --max_write_size=104857600 --bitmap=1"
+    cmd_opts = "--min_write_size=10485760 --max_write_size=80857600 --bitmap=1"
```

Comment:
why `80857600`, is it upper limit of total nblks in the portion ?

Reply:
yes, it is a little smaller than the upper limit.

Reply:
JFYI:  it translate to 2.4TB if blk size = 4K when storing bit map:) 

But maybe we are looking for 10x of this or even more.

Reply:
Thanks @xiaoxichen for bring it up. Yes, my thought is to create only one chunk for meta blk store, and change chunkz-size to uint64_t which should be big enough. 

Another thing is metablkstore has compression feature so the bitmap can be significantly compressed before written.

Reply:
ok if it is per allocator then we are good?

Reply:
yes, we are good as in nublox bitmap.